kola/tests/misc/selinux.go

```diff
@@ -40,6 +43,15 @@ func init() {
 		Distros:     []string{"cl", "fcos", "rhcos"},
 		// This test is normally not related to the cloud environment
 		Platforms: []string{"qemu", "qemu-unpriv"},
+		SkipFunc: func(version semver.Version, channel, arch, platform string) bool {
+			// Workaround to set the SELinux boolean name based of the Flatcar version.
+			// Note: it works only if we test '*'
+			if version.LessThan(semver.Version{Major: 3637}) {
```

Comment:
Should this be 3733?

```suggestion
			if version.LessThan(semver.Version{Major: 3733}) {
```